### PR TITLE
l10n: uk: update translation

### PR DESCRIPTION
**Pull request by arkid15r**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -10269,85 +10269,87 @@ msgstr "порожній mbox: \"%s\""
...
 
 #: builtin/merge-base.c:154
 msgid "is the first one ancestor of the other?"
-msgstr ""
+msgstr "чи є перша з них предком другої?"
 
 #: builtin/merge-base.c:156
 msgid "find where <commit> forked from reflog of <ref>"
-msgstr ""
+msgstr "знайти, де <коміт> відгалузився від логу посилань <посилання>"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "знайти, де <коміт> відгалузився від журналу посилань <посилання>"
```

File: po/uk.po [hunk 2]
```diff
@@ -10269,85 +10269,87 @@ msgstr "порожній mbox: \"%s\""
...
 
 #: builtin/merge-file.c:39
 msgid "use a diff3 based merge"
-msgstr ""
+msgstr "використовувати злиття на основі diff3"
 
 #: builtin/merge-file.c:40
 msgid "use a zealous diff3 based merge"
-msgstr ""
+msgstr "використовуйте ретельне злиття на основі diff3"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "використовувати ретельне злиття на основі diff3"
```

File: po/uk.po [hunk 3]
```diff
@@ -10363,13 +10365,13 @@ msgstr "не вдалося розібрати обʼєкт \"%s\""
 #, c-format
 msgid "cannot handle more than %d base. Ignoring %s."
 msgid_plural "cannot handle more than %d bases. Ignoring %s."
-msgstr[0] ""
-msgstr[1] ""
-msgstr[2] ""
+msgstr[0] "неможливо обробити більш ніж %d бази. Ігнорування %s."
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr[0] "неможливо обробити більш ніж %d базу. Ігнорування %s."
```

File: po/uk.po [hunk 4]
```diff
@@ -10363,13 +10365,13 @@ msgstr "не вдалося розібрати обʼєкт \"%s\""
...
+msgstr[1] "неможливо обробити більш ніж %d баз. Ігнорування %s."
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr[1] "неможливо обробити більш ніж %d бази. Ігнорування %s."
```

File: po/uk.po [hunk 5]
```diff
@@ -10379,63 +10381,63 @@ msgstr "не вдалося розвʼязати посилання \"%s\""
...
 
 #: builtin/merge-tree.c:524
 msgid "do a real merge instead of a trivial merge"
-msgstr ""
+msgstr "зробити справжнє злиття замість тривіального злиття"
 
 #: builtin/merge-tree.c:527
 msgid "do a trivial merge only"
-msgstr ""
+msgstr "робити лише тривіальне злиття"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "зробити лише тривіальне злиття"
```

**Review by kasya — COMMENTED** [review]
LGTM!

File: po/uk.po (hunk 1 above)
Comment (arkid15r) [reply po/uk.po]: 👍 consistency!

**Pull request merged**